src/js/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,29 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import InputControl from '../InputControl'
+
+class Checkbox extends PureComponent {
+  render() {
+    return <InputControl type='checkbox' { ...this.props } />
+  }
+}
+
+Checkbox.propTypes = {
+  name: PropTypes.string.isRequired,
+  label: PropTypes.string.isRequired,
+  inputProps: PropTypes.object,
+  isInline: PropTypes.bool,
+  className: PropTypes.string,
+  mods: PropTypes.string,
+  style: PropTypes.object
```

Comment:
`mods` are modifiers? What's the difference between adding additional class names to `className` and adding something to `mods`? I don't quite understand that.

Reply:
So in this case it helps save you the understanding of what the base class needs to be.  So for an example of say button, the default class is 'Button'.  The following two will allow for the same outputs, except in one case you don't need to rewrite or even know about the base class.  `<Button className='Button someUniqueModifier' />` and `<Button mods='someUniqueModifier' />`